python/lsst/jointcal/jointcal.py

```diff
@@ -284,47 +301,47 @@ def __init__(self, config, sourceFluxField, maxMag, centroid, shape):
         self.config = config
         self.sourceFluxField = sourceFluxField
         self.maxMag = maxMag
-        self.centroid=centroid
-        self.shape=shape
+        self.centroid = centroid
+        self.shape = shape
 
     def select(self, srcCat, calib):
-# Return a catalog containing only reasonnable stars / galaxies
+        """Return a catalog containing only reasonnable stars / galaxies."""
 
         schema = srcCat.getSchema()
         newCat = afwTable.SourceCatalog(schema)
         fluxKey = schema[self.sourceFluxField+"_flux"].asKey()
         fluxErrKey = schema[self.sourceFluxField+"_fluxSigma"].asKey()
         parentKey = schema["parent"].asKey()
         flagKeys = []
-        for f in self.config.badFlags :
+        for f in self.config.badFlags:
             key = schema[f].asKey()
             flagKeys.append(key)
         fluxFlagKey = schema[self.sourceFluxField+"_flag"].asKey()
         flagKeys.append(fluxFlagKey)
 
-        for src in srcCat :
+        for src in srcCat:
             # Do not consider sources with bad flags
-            for f in flagKeys :
+            for f in flagKeys:
                 rej = 0
-                if src.get(f) :
+                if src.get(f):
                     rej = 1
                     break
-            if rej == 1 :
+            if rej == 1:
                 continue
             # Reject negative flux
             flux = src.get(fluxKey)
-            if flux < 0 :
+            if flux < 0:
                 continue
             # Reject objects with too large magnitude
             fluxErr = src.get(fluxErrKey)
             mag, magErr = calib.getMagnitude(flux, fluxErr)
-            if mag > self.maxMag or magErr > 0.1 or flux/fluxErr < 10 :
+            if mag > self.maxMag or magErr > 0.1 or flux/fluxErr < 10:
```

Comment:
Please eliminate the hard-coded values by making them config parameters or arguments (in this ticket or file a new one if you'd rather not fix it right now).